pkg/config/config.go

```diff
@@ -162,6 +162,7 @@ type AdapterConfig interface {
 	BlockstoreAzureParams() (blockparams.Azure, error)
 	BlockstoreExtras() map[string]string
 	GetDefaultNamespacePrefix() *string
+	SigningKey() string
```

Comment:
How about `BlockstoreSigningKey`?

(Or when thinking about, and it's out-of-scope for this PR -
WDYT about removing the `Blockstore` prefix from all these param names?)

Reply:
If we should do anything is probably remove the Blockstore prefix.
However since it is not related to the PR I suggest we do that in a separate issue

Reply:
So can you please open an Issue for it?

Reply:
Feel free to open an issue for this